### PR TITLE
MGMT-1620 - preparation for 4.6: extract openshift-baremetal-installe…

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**
…r from OCP release and pull secret from install-config

### CODE REVIEW

File: render_files.py [hunk 1]
```diff
@@ -50,7 +51,10 @@ def update_bmh_files(ignition_file, cluster_id, inventory_endpoint):
             data = json.load(file_obj)
             storage_files = data['storage']['files']
             # since we don't remove file for now, we don't need to iterate through copy
+            # for file_data in storage_files[:]:
```
Comment (tsorya) [inline render_files.py:54]: comments are for future use?

File: render_files.py [hunk 2]
```diff
@@ -119,12 +125,28 @@ def prepare_install_config(config_dir, install_config):
             file_obj.write(install_config)
 
 
+def set_pull_secret(config_dir):
+    with open(os.path.join(config_dir, INSTALL_CONFIG), 'r') as json_file:
+        pull_secret = yaml.safe_load(json_file)['pullSecret']
+    with open('/root/.docker/config.json', 'w+') as config_file:
+        config_file.write(pull_secret)
```
Comment (tsorya) [inline render_files.py:132]: why not json.dump?

File: render_files.py [hunk 3]
```diff
@@ -119,12 +125,28 @@ def prepare_install_config(config_dir, install_config):
...
 def create_config_dir(work_dir):
     config_dir = os.path.join(work_dir, "installer_dir")
     subprocess.check_output(["mkdir", "-p", config_dir])
     return config_dir
 
 
+def extract_baremetal_installer(work_dir, openshift_release_image):
+    try:
+        command = "%s/oc adm release extract --command=openshift-baremetal-install  --to=%s %s" \
+                % (work_dir, work_dir, openshift_release_image)
```
Comment (tsorya) [inline render_files.py]: use dict %{work_dir}s

**Comment by tsorya** [comment]
small comments

File: render_files.py (hunk 1 above)
Comment (yevgeny-shnaidman) [reply render_files.py:54]: yes, this is preparations for 4.6 in 2 weeks

File: render_files.py (hunk 2 above)
Comment (yevgeny-shnaidman) [reply render_files.py:132]: because install-config is in yaml format, and not json

File: render_files.py (hunk 3 above)
Comment (yevgeny-shnaidman) [reply render_files.py]: fixed

File: render_files.py (hunk 2 above)
Comment (tsorya) [reply render_files.py:132]: so yaml.dump

Comment (tsorya) [reply render_files.py:132]: but u call file json
so how to are saving yaml there?

**Review by tsorya — APPROVED** [review]
change it to yaml dump

**Pull request merged**